redirects/13.0.txt

```diff
@@ -167,7 +167,8 @@ developer/reference/orm.rst                               developer/reference/ad
 developer/reference/reports.rst                           developer/reference/addons/reports.rst
 developer/reference/security.rst                          developer/reference/addons/security.rst
 developer/reference/testing.rst                           developer/reference/addons/testing.rst
-developer/reference/views.rst                             developer/reference/addons/views.rst
```

Comment:
The page is moved in master; therefore you should add the redirect rules in a new `redirects/saas-16.4.txt` file.